static/css/tools/statvar_menu.scss

```diff
@@ -52,6 +52,19 @@ $menu-line: #5b5e60;
   padding: 24px;
 }
 
+#drill .noborder{
```

Comment:
Nit: please ad a space before {
And remove "px" if the value is 0

Reply:
done